R/getPrevalence.R

```diff
@@ -629,18 +629,23 @@ setGeneric("agglomerateByPrevalence", signature = "x",
 #' @rdname agglomerate-methods
 #' @export
 setMethod("agglomerateByPrevalence", signature = c(x = "SummarizedExperiment"),
-    function(x, rank = taxonomyRanks(x)[1L], other_label = "Other", ...){
```

Comment:
Sorry, I did not notice that agglomerateByPrevalence has also rank parameter. In this case, things get even more simpler.

--> Just change the default value of rank to NULL. It should work --> test